docs/source/tutorial-beginner.rst

```diff
@@ -21,17 +21,17 @@ The Hello World program is a good example of the basic pieces used in any Cozmo
 The code for the Hello World program looks like this.
 
 .. code-block:: python
-  :lineno-start: 17
+  :lineno-start: 1
```

Comment:
This needs to match the hello_world.py program (so with the docstring before the imports), I had the line nos start at 17 rather than paste in the top 16 lines of copyright information.